samples/apps/copilot-chat-app/webapi/Storage/ChatMessageRepository.cs

```diff
@@ -28,7 +28,8 @@ public Task<IEnumerable<ChatMessage>> FindByChatIdAsync(string chatId)
 
     public async Task<ChatMessage> FindLastByChatIdAsync(string chatId)
     {
-        var messages = await this.FindByChatIdAsync(chatId);
+        var chatMessages = await this.FindByChatIdAsync(chatId);
```

Comment:
Same here.  This can be:
```C#
var chatMessages = await this.FindByChatIdAsync(chatId);
var first = chatMessages.OrderByDescending(e => e.Timestamp).FirstOrDefault();
if (first is null)
{
    throw new KeyNotFoundException($"No messages found for chat {chatId}.");
}
return first;
```

Reply:
what about using MaxBy?

Reply:
You'd need to be targeting .NET 6+.

Reply:
Ah, ok, I had it in my head the whole repo was targeting netstandard2.0.